Week-1/index.css

```diff
@@ -0,0 +1,113 @@
+
+body{
+    font-family: 'Lato', sans-serif;
+}
+
+.main-container{
+    display: grid;
+}
+
+table{
+    width: 100%;
```

Comment:
Lets add class to table and add styles to class selector..becoz if u add other table in ur html then ur code might break..